test-bench-utils/README.md

```diff
@@ -18,12 +18,13 @@ Then create a file under _lib/sinks/_ that exports functions with a consistent
 signature:
 ```js
   /**
-   * @param {string} input user input string
+   * @param {Object} params
+   * @param {string} params.input user input string
```

Comment:
should we call out you can pass in whatever and it's derived from the lib/routes.js?

Reply:
i'll improve these docs!